zulip_bots/zulip_bots/bots/followup/test_followup.py

```diff
@@ -3,29 +3,34 @@
 from __future__ import absolute_import
 from __future__ import print_function
 
-from zulip_bots.test_lib import BotTestCase
+from zulip_bots.test_lib import (
+    StubBotHandler,
+    StubBotTestCase,
+    get_bot_message_handler,
+)
 
 from typing import Any
 
-class TestFollowUpBot(BotTestCase):
+
+class TestFollowUpBot(StubBotTestCase):
     bot_name = "followup"
 
-    def test_bot(self) -> None:
-        expected_send_reply = [
-            ("", 'Please specify the message you want to send to followup stream after @mention-bot')
-        ]
-        self.check_expected_responses(expected_send_reply, expected_method='send_reply')
-
-        expected_send_message = [
```

Comment:
I think we want to keep the spirit of this test, which is that it handles the "normal" case.

Reply:
I have handled this case in test_no_text()

Reply:
What about below?..."I have completed my task"